tests/src/tools/ReadyToRun.SuperIlc/ReadyToRun.SuperIlc.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <!-- <TargetName>SuperIlc</TargetName> -->
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <NoWarn>$(NoWarn);NU1701</NoWarn>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="System.CommandLine" Version="0.1.0-e160119-1" />
```

Comment:
This CoreFX command line package is a dead-end. 

I think we may want to start using https://github.com/dotnet/command-line-api so that we can provide feedback; and save us some time migrating away from the dead-end package later.

Reply:
Done. I have some feedback on ease of use out of the box that I'll share with them.